new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInPV.java

```diff
@@ -295,38 +308,111 @@ public void testDomainInPvUsingWlst() {
                         .nodePort(0))
                     .addChannelsItem(new Channel()
                         .channelName("T3Channel")
-                        .nodePort(t3ChannelPort))))
+                        .nodePort(wdtT3ChannelPort))))
             .addClustersItem(new Cluster() //cluster
-                .clusterName(clusterName)
-                .replicas(replicaCount)
+                .clusterName(wdtClusterName)
+                .replicas(wdtReplicaCount)
                 .serverStartState("RUNNING")));
 
     // verify the domain custom resource is created
-    createDomainAndVerify(domain, wlstDomainNamespace);
+    createDomainAndVerify(domain, wdtDomainNamespace);
 
     // verify admin server pod is ready
-    checkPodReady(adminServerPodName, domainUid, wlstDomainNamespace);
+    checkPodReady(wdtAdminServerPodName, wdtDomainUid, wdtDomainNamespace);
 
     // verify the admin server service created
-    checkServiceExists(adminServerPodName, wlstDomainNamespace);
+    checkServiceExists(wdtAdminServerPodName, wdtDomainNamespace);
 
     // verify managed server pods are ready
-    for (int i = 1; i <= replicaCount; i++) {
+    for (int i = 1; i <= wdtReplicaCount; i++) {
       logger.info("Waiting for managed server pod {0} to be ready in namespace {1}",
-          managedServerPodNamePrefix + i, wlstDomainNamespace);
-      checkPodReady(managedServerPodNamePrefix + i, domainUid, wlstDomainNamespace);
+          wdtManagedServerPodNamePrefix + i, wdtDomainNamespace);
+      checkPodReady(wdtManagedServerPodNamePrefix + i, wdtDomainUid, wdtDomainNamespace);
     }
 
     // verify managed server services created
-    for (int i = 1; i <= replicaCount; i++) {
+    for (int i = 1; i <= wdtReplicaCount; i++) {
       logger.info("Checking managed server service {0} is created in namespace {1}",
-          managedServerPodNamePrefix + i, wlstDomainNamespace);
-      checkServiceExists(managedServerPodNamePrefix + i, wlstDomainNamespace);
+          wdtManagedServerPodNamePrefix + i, wdtDomainNamespace);
+      checkServiceExists(wdtManagedServerPodNamePrefix + i, wdtDomainNamespace);
     }
+    previousTestSuccessful = true;
+  }
+
+  /**
+   * Deploy an application using admin t3 channel port, target to both cluster and admin server.
+   */
+  @Test
+  @Order(2)
+  @DisplayName("Deploy an application using t3 channel port")
+  public void testDeployAppUsingT3ChannelPort() {
```

Comment:
If the actual test is verifying the domain can be created/started using domain in PV and WDT. It should only have the checks to make sure the domain is created/started IMO. If I make this as an assertion in the same domain creation test, if this assertion fails for some reason, it will mark the domain creation test as failed which is actually not correct.

Reply:
IMO - its setup, verify (single test)
This method  `testDeployAppUsingT3ChannelPort` can be used in any IT class, one may may fail and other may pass depending on the domain setup/startup. 
That's what we used in JUnit4 as well.

Reply:
domain could have been created/started successfully, but application deployment may fail, they are two different tests.

Reply:
moved app deployment to the actual test and leaving other use cases in individual tests..waiting for jenkins results..